static_src/test/unit/stores/user_store.spec.js

```diff
@@ -134,22 +135,25 @@ describe('UserStore', function () {
       expectedUsers = [
         {
           guid: userGuidA,
-          roles: { [spaceGuid]: ['space_developer'] }
+          roles: { [spaceGuid]: ['space_developer'] },
+          space_roles: [ 'space_developer' ]
```

Comment:
ahh didn't know it was already there. in that case, we can leave it. i think we need to revisit the whole topic of the post-processing of data (like this transformation). but another day!